contrib/completion/bash/docker

```diff
@@ -1441,6 +1445,99 @@ _docker_version() {
 	esac
 }
 
+_docker_volume_create() {
+	case "$prev" in
+		--driver|-d)
+			COMPREPLY=( $( compgen -W "local" -- "$cur" ) )
```

Comment:
Should I add the plugin drivers mentioned in https://github.com/docker/docker/blob/master/docs/extend/plugins.md here? WDYT?